packages/material-ui-lab/src/Timeline/Timeline.tsx

```diff
@@ -48,20 +40,71 @@ export interface TimelineProps extends StandardProps<React.HTMLAttributes<HTMLUL
     /** Styles applied to the root element if `align="alternate"`. */
     alignAlternate?: string;
   };
+
+  /**
+   * className applied to the root element.
+   */
+  className?: string;
```

Comment:
Thus is weird, I was expecting it to already come for free with `React.HTMLAttributes<HTMLUListElement`